packages/xfce4_dev_tools.rb

```diff
@@ -0,0 +1,21 @@
+require 'package'
+
+class Xfce4_dev_tools < Package
+    description 'Xfce4 development tools'
+    homepage "https://xfce.org/"
+    version '4.14'
+    compatibility 'all'
+    source_url "https://archive.xfce.org/src/xfce/xfce4-dev-tools/#{version}/xfce4-dev-tools-#{version}.0.tar.bz2"
```

Comment:
I prefer to not use `#{version}` in urls.  It makes it easier to `sha256sum` and cut and paste.